include/ibamr/INSHierarchyIntegrator.h

```diff
@@ -203,9 +203,18 @@ class INSHierarchyIntegrator : public IBTK::HierarchyIntegrator
 
     /*!
      * Supply a fluid source/sink distribution.
+     *
+     * @deprecated Use registerVelocityDivergenceFunction() instead.
      */
     void registerFluidSourceFunction(SAMRAI::tbox::Pointer<IBTK::CartGridFunction> Q_fcn);
 
+    /*!
+     * Add a CartGridFunction that computes \f$ \nabla \cdot \mathbf{u} \f$.
```

Comment:
For consistency with other function docs, suggest:
```suggestion
     * Supply a CartGridFunction that specifies \f$ \nabla \cdot \mathbf{u} = Q\f$.
```